File: CS/PieTotalLabelSample/Form1.cs
using DevExpress.XtraCharts;
using System;
using System.Windows.Forms;

namespace PieTotalLabelSample {
    public partial class Form1 : Form {
        public Form1() {
            InitializeComponent();
        }

        private void Form1_Load(object sender, EventArgs e) {
            #region #PieTotalLabel
            PieTotalLabel totalLabel = ((DoughnutSeriesView)pieChart.Series["Country Areas"].View).TotalLabel;
            totalLabel.Visible = true;
            totalLabel.TextPattern = "Total area\n{TV:F2} km\xB2";
            #endregion #PieTotalLabel
        }
    }
}

File: VB/PieTotalLabelSample/Form1.vb
Imports DevExpress.XtraCharts
Imports System
Imports System.Windows.Forms

Namespace PieTotalLabelSample
    Partial Public Class Form1
        Inherits Form

        Public Sub New()
            InitializeComponent()
        End Sub

        Private Sub Form1_Load(ByVal sender As Object, ByVal e As EventArgs) Handles Me.Load
'            #Region "#PieTotalLabel"
            Dim totalLabel As PieTotalLabel = CType(pieChart.Series("Country Areas").View, DoughnutSeriesView).TotalLabel
            totalLabel.Visible = True
            totalLabel.TextPattern = "Total area" & vbLf & "{TV:F2} km" & ChrW(&HB2).ToString()
'            #End Region ' #PieTotalLabel
        End Sub
    End Class
End Namespace
